janus_core/cli/phonons.py

```diff
@@ -55,16 +55,28 @@ def phonons(
     ] = False,
     temp_start: Annotated[
         float,
-        Option(help="Start temperature for CV calculations, in K."),
+        Option(help="Start temperature for thermal properties calculations, in K."),
     ] = 0.0,
     temp_end: Annotated[
         float,
-        Option(help="End temperature for CV calculations, in K."),
+        Option(help="End temperature for thermal properties calculations, in K."),
     ] = 1000.0,
     temp_step: Annotated[
         float,
-        Option(help="Temperature step for CV calculations, in K."),
+        Option(help="Temperature step for thermal properties calculations, in K."),
     ] = 50,
+    band: Annotated[
+        bool,
+        Option(help="Whether to save fc in hdf5."),
+    ] = False,
+    hdf5: Annotated[
+        bool,
+        Option(help="Whether to calculate the band structure."),
+    ] = True,
```

Comment:
Are these descriptions backwards?

Reply:
yes I need to updated them... copy paste...